SPECIFICATION.md

```diff
@@ -101,6 +101,16 @@ message_hash = sha256(serialized_payload)
 The serialized payload must follow the `Streamable` standard defined
 [here](https://github.com/Chia-Network/chia-blockchain/blob/main/chia/util/streamable.py).
 
+## Pool URL
+The pool URL is the url that farmers use to connect to the pool. The subdomains, port, and path are optional. The client
+will use 443 if there is no port. Note that the trailing slash must NOT be present.
+```
+https://subdomain.domain.tld/path
+OR 
+https://subdomain.tld:port/path
```

Comment:
```suggestion
https://domain.tld:port/path
```

no?

Reply:
actually I meant to use `subdomain.domain` in both